src/main/java/seedu/duke/data/ShoppingList.java

```diff
@@ -10,6 +10,10 @@
     public ShoppingList(){
     }
 
+    public static void resetBudget() {
+        budget = 0.0;
+    }
```

Comment:
Perhaps this function should be in the Budget class and not the ShoppingList class.

Reply:
Thanks!!! Will make the changes!!